tests/e2e/e2e_test.go

```diff
@@ -188,31 +188,36 @@ var _ = Describe("Usecase #5", func() {
 			// Push the changes
 			runCmd("ocdev push --local " + tmpDir + "/nodejs-ex")
 
-			// ping the ip
-			pingSvc(pingUrl)
+			// Create route
+			runCmd("ocdev url create nodejs")
+			getRoute := runCmd("ocdev url list  | sed -n '1!p' | awk '{ print $3 }'")
+			getRoute = strings.TrimSpace(getRoute)
+			Ω(getRoute).Should(ContainSubstring("nodejs-" + projName))
```

Comment:
what is Ω ?

Reply:
@kadel ginkgo/geomega notation

Reply:
@ashetty1 , how to type that (/me curious) ?

Reply:
it looks really strange. Is there an alternative  to using  Ω? Something that is a little bit more standard?